src/components/Ingredients/Ingredients.test.js

```diff
@@ -0,0 +1,168 @@
+import { getByTestId, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import IngredientForm from './IngredientForm';
+import Ingredient from './Ingredient';
+import Ingredients from './Ingredients';
+import * as redux from 'react-redux';
+import * as actions from '../../store/recipes/actions';
+import { Provider } from 'react-redux';
+import store from '../../store/store';
+
+it('empty IngredientForm renders when button clicked', async () => {
+  render(
+    <Provider store={store}>
+      <Ingredients isEditMode={true} />
+    </Provider>
+  );
+  const addIngredientBtn = await waitFor(() => screen.getByTestId('add-ingredient-btn'));
+  userEvent.click(addIngredientBtn);
+
+  expect(screen.getByTestId('form')).toBeInTheDocument();
+});
+
+it('filled IngredientForm renders when button clicked', async () => {
+  render(
+    <Provider store={store}>
+      <Ingredient ingredientId={2} amount={4} unit={'szt'} name={'bananas'} isEditMode={true} />
+    </Provider>
+  );
+  const editButton = await waitFor(() => screen.getByTestId('edit-btn-2'));
+  userEvent.click(editButton);
+
+  expect(screen.getByTestId('form')).toBeInTheDocument();
```

Comment:
Jeżeli test ma sprawdzać, czy `filled IngredientForm renders`, to warto by też sprawdzić zawartość pól formularza ☝️